### PR TITLE
use shared_image to make changing image faster and comment out placem…

### PR DESCRIPTION
**Pull request by stephenstubbs**
…ent constraints.

### CODE REVIEW

File: docker-compose.image.yml [hunk 1]
```diff
@@ -1,11 +1,16 @@
-version: "3"
+version: "3.7"
```
Comment (tiredpixel) [inline docker-compose.image.yml:1]: There is nothing in this patch which depends on Compose 3.7. But I note I've done similarly for the other file, and I don't really care—just pointing it out.

**Review by tiredpixel — APPROVED** [review]
Thanks. :)

File: docker-compose.image.yml (hunk 1 above)
Comment (stephenstubbs) [reply docker-compose.image.yml:1]: 3.7 is just for the shared image which I've added. 3 doesnt have this functionality. Just saves having to add the image name multiple times. Not needed but definitely a time saver forn testing multiple images.

**Comment by tiredpixel** [comment]
Kindly note 76ce19a , which has updated LICENSE and README to link to a page noting your handle. Also please note the commit message, which mentions that I don't think linking to every contributor who is not a maintainer or has at least a commit-bit is maintainable—but nevertheless, many thanks for your contribution, and your copyright is noted appropriately. :)

File: docker-compose.image.yml (hunk 1 above)
Comment (tiredpixel) [reply docker-compose.image.yml:1]: Oh, is that so? I didn't realise. I thought your patch had purely YAML anchor changes, which has nothing to do with Docker Compose, only the YAML parser used pre-validation. But perhaps I missed something. :)

**Pull request merged**